tests/run/numpy_test.pyx

```diff
@@ -255,7 +255,7 @@ def inc1_object(np.ndarray[object] arr):
     arr[1] = o # unfortunately, += segfaults for objects
 
 def inc1_int_t(np.ndarray[np.int_t] arr):               arr[1] += 1
-def inc1_long_t(np.ndarray[np.long_t] arr):             arr[1] += 1
+# def inc1_long_t(np.ndarray[np.long_t] arr):             arr[1] += 1
```

Comment:
```suggestion
```

I'd just delete it if its no longer relevant. (Again, I'll do this just before merging, so no need to re-run the CI for this)